### PR TITLE
Add monitoring section in application management

### PR DESCRIPTION
**Pull request by aramalipoor**

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -44,6 +44,7 @@
   * [Cloning resources](management/cloning-resources.md)
   * [Deploying existing images](management/deploying-existing-images.md)
   * [Logs](management/logs.md)
+  * [Monitoring](management/monitoring.md)
```
Comment (alikhajeh1) [inline SUMMARY.md]: Can we call this `Application Performance Monitoring` so it's not confused with the cpu/memory/network metrics and Monitoring menu (shows recent events) that exists in AbarCloud already.

File: management/monitoring.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Monitoring
```
Comment (alikhajeh1) [inline management/monitoring.md]: https://github.com/abarcloud/docs/pull/44/files#r153852128

File: management/monitoring.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+We strongly recommend installing application-level monitoring to your projects.
```
Comment (alikhajeh1) [inline management/monitoring.md]: We strongly recommend you use application performance monitoring (APM) in your projects.

File: management/monitoring.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+We strongly recommend installing application-level monitoring to your projects. 
+Components of a complete application performance management solution includes
+performance of individual web requests or transactions, application dependencies like databases,
+web services, caching, code level performance profiling and applications errors monitoring. 
+
+There are many APM solutions available:
+* [New Relic APM](https://newrelic.com/application-monitoring)
+* [Datadog](https://www.datadoghq.com/)
+* [AppDynamics](https://www.appdynamics.com/php/)
```
Comment (alikhajeh1) [inline management/monitoring.md]: Maybe link to https://www.appdynamics.com/product/application-performance-management/ users can see they support multiple languages

File: management/monitoring.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+## Newrelic APM
+To install and enable Newrelic's application monitoring follow these steps:
+
+1. Create an account in [Newrelic](https://newrelic.com/signup).
+2. Get your **license key** in application setup wizard.
+3. Add these 2 environment variables to your application's **BuildConfig**
+    * `NEW_RELIC_LICENSE_KEY`: *License key acquired through Newrelic setup wizard.*
+    * `NEW_RELIC_APP_NAME`: My Application
+4. Install Newrelic agent's in your Dockerfile, according to your application's language.
```
Comment (alikhajeh1) [inline management/monitoring.md]: `New relic`

File: management/monitoring.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+Below is a PHP (5 & 7) example you can add to your Dockerfile:
```
Comment (alikhajeh1) [inline management/monitoring.md]: If you used our [PHP 5 or 7 example Dockerfile](../quickstart/dockerfile.md), you can add the following changes:

#### Dockerfile
docco link to Dockerfile snippet from examples repo

#### supervisord.conf
docco link to supervisord.conf snippet from examples repo

File: SUMMARY.md (hunk 1 above)
Comment (aramalipoor) [reply SUMMARY.md]: You're right but this title is too long. I want something short and clean. What about  `Application Monitoring` or `Performance Monitoring`

Comment (alikhajeh1) [reply SUMMARY.md]: Application Monitoring is good! You can then have "Performance monitoring" and "Exception monitoring" to it (to mention Sentry) and remove https://docs.abarcloud.com/management/best-practices.html#monitor-performance-and-exceptions

File: management/monitoring.md (hunk 4 above)
Comment (aramalipoor) [reply management/monitoring.md]: Oops! My bad, ought to do that

**Pull request merged**